h/schemas/analytics.py

```diff
@@ -10,7 +10,7 @@ class EventSchema(JSONSchema):
         "properties": {
             "event": {
                 "type": "string",
-                "enum": ["APPLY_PENDING_UPDATES"],
+                "enum": ["client.realtime.apply_updates"],
```

Comment:
Considering the client is the only official consumer for h, the `client` prefix might be considered redundant, but it is possible that at some point, different backend services (lms, via, h...) end up sending events to a shared system. In that case the prefix will come handy.